src/main/Extensions/PasswordGenerator/PasswordGeneratorExtension.ts

```diff
@@ -0,0 +1,220 @@
+import type { AssetPathResolver } from "@Core/AssetPathResolver";
+import type { Extension } from "@Core/Extension";
+import type { SettingsManager } from "@Core/SettingsManager";
+import {
+    createCopyToClipboardAction,
+    createEmptyInstantSearchResult,
+    type InstantSearchResultItems,
+    type SearchResultItem,
+} from "@common/Core";
+import { getExtensionSettingKey } from "@common/Core/Extension";
+import type { Image } from "@common/Core/Image";
+import type { PasswordGeneratorSettings as Settings } from "@common/Extensions/PasswordGenerator";
+import { PasswordGeneratorDefaultSymbols } from "@common/Extensions/PasswordGenerator/PasswordGeneratorDefaultSymbols";
```

Comment:
You can simplify this import:

```suggestion
import { PasswordGeneratorDefaultSymbols } from "@common/Extensions/PasswordGenerator";
```